deploy/app.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bot-api
+  namespace: kscoutbot
```

Comment:
don't specify namespace in here, specify when we deploy with `kubectl -n SPECIFY_NAMESPACE_HERE` so that it is more flexible in the future.